common-components/index.md

```diff
@@ -399,66 +399,39 @@ If you're considering your options and want more advice, get in touch with your
+- We are also looking to put in place contracts with suppliers which the SDT will manage to ensure that they build using Departmental best practise and these can be utilised to spin up a full scrum team.
 
-__Support__
+With all of these delivery routes the SDT will be ensuring systems are built to a standard that has been agreed internally. We are able to reuse solutions and ensure systems are built in a similar way which makes ongoing maintenance and support easier to handle.
 
-The Enterprise CRM is supported internally by a central team within CIPS.  The team uses the ServiceNow toolset to manage the support of this service and follow DfE’s central service management processes.
+We have a standard support model for systems built using these standards.
 
-To report an issue with the service, log a ticket via the [Service Portal](https://dfe.service-now.com/serviceportal). The team are available from 08:00 – 22:00 hrs Monday to Friday, excluding UK public holidays. If the service desk is unable to resolve your query, the issue will be passed to a specialist technical team.  
+At a high level this is:
 
-__Roadmap__
-
-DfE customers already using the Enterprise CRM or undergoing a transition to adopt the service include: 
-- National Apprenticeship Service (267 users) 
-- ESFA Customer Service Team (1076 users)
-
-The Enterprise CRM is tailored to meet unique customer requirements in every new deployment. A range of enhanced solution features are also planned, these include:
-- Outlook integration
-- Document storage
-- Data analysis
-- Notifications
-- Integrated reporting
-- AI builder
-- Customer relationship insights
-
-__Further guidance__
+- SLAs for incidents and requests aligned to wider Technology directorate operational levels.
+- Incidents and requests handled through DfE’s ITSM Tool ServiceNow.
+- For ongoing work – A set amount of development days a year based off the criticality and complexity of your system to support work requests, along with support in scoping and sizing the change requests to maximise the development time you have available. 
+- Our team working closely with service delivery management to ensure service standards are maintained and ongoing service governance is met such as AtO and ITHCs.
 
-Contact [David Bowley](https://eur.delve.office.com/?u=2f894eab-9154-4025-9a34-4d6236a07b08&v=profiledetails) to find out more about this solution.
+The SDT have also set up a Dynamics Community of Practise with other teams in the Department who are utilising D365 in their projects. This is to ensure ongoing conversations are taking place to standardise across the Department and ensure best practise is shared.
 
+If anyone is interested in chatting about options please contact: [Iain McIntosh](https://gbr.delve.office.com/?u=b7680d65-eb71-4c1c-a68e-b95fa6248026&v=work).
```

Comment:
Is there a mailbox or Slack / Teams channel we can link to? Trying to reduce the amount of named contacts, for security / future-proofing reasons.

Reply:
Not yet, will be in 2/3 weeks time, future update